packages/core/src/components/Modal/Popup/Popup.styled.tsx

```diff
@@ -49,6 +48,7 @@ export const Popup = styled('div')<ModalPopupProps>`
     box-sizing: border-box;
     border-top-left-radius: 1.6rem;
     border-top-right-radius: 1.6rem;
+    ${({ overflowVisible }) => !overflowVisible && `overflow: hidden;`};
```

Comment:
You can write like this overflow: ${({ overflowVisible }) => !overflowVisible && 'hidden'}; for better readability.